packages/decorator/meta.yaml

```diff
@@ -0,0 +1,7 @@
+package:
+  name: decorator
+  version: 4.3.0
+
+source:
+  url: https://files.pythonhosted.org/packages/6f/24/15a229626c775aae5806312f6bf1e2a73785be3402c0acdec5dbddd8c11e/decorator-4.3.0.tar.gz
+  sha256: c39efa13fbdeb4506c476c9b3babf6a718da943dab7811c206005a4a956c080c
```

Comment:
Please test the import, see e.g. https://github.com/conda-forge/decorator-feedstock/blob/master/recipe/meta.yaml

Reply:
I'm not sure what you mean by that (sorry, I'm a bit of a python noob). I verified that the networkx package worked in an electron-based pyodide experiment of mine

Reply:
I meant to add a `test` section to the `meta.yaml`,
```
test:
  imports:
    - decorator
```

same as you did in the other package to check that the import works as expected.

Reply:
Ahh, gotcha